GameFiles/IW4x/userraw/scripts/_integration.gsc

```diff
@@ -574,6 +586,9 @@ OnExecuteCommand( event )
                 response = self GotoImpl( data );
             }
             break;
+        case "PlayerToMe":
+            response = self PlayerToMeImpl( event.target );
```

Comment:
Because the action is being taking on the target, this should be switched too

`event.target self PlayerToMeImpl( self )`